plugins/generic/announcementFeed/templates/rss.tpl

```diff
@@ -28,7 +28,7 @@
 
 		{* optional elements *}
 		{if $journal->getPrimaryLocale()}
-		<dc:language>{$journal->getPrimaryLocale()|replace:'_':'-'|strip|escape:"html"}</dc:language>
+		<dc:language>{$language|escape}</dc:language>
```

Comment:
As the `$language` doesn't depend on the `$context->getPrimaryLocale()` (which I guess should be always available anyway), then the `{if}` can be removed.